nucypher/utilities/porter/porter.py

```diff
@@ -0,0 +1,138 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+from constant_sorrow.constants import NO_CONTROL_PROTOCOL
+from flask import request, Response
+
+from nucypher.blockchain.eth.registry import BaseContractRegistry, InMemoryContractRegistry
+from nucypher.control.controllers import WebController, JSONRPCController
+from nucypher.network.nodes import Learner
+from nucypher.utilities.logging import Logger
+from nucypher.utilities.porter.control.controllers import PorterCLIController
+from nucypher.utilities.porter.control.interfaces import PorterInterface
+
+
+class Porter(Learner):
+
+    BANNER = r"""
+
+ ______
+(_____ \           _
+ _____) )__   ____| |_  ____  ____
+|  ____/ _ \ / ___)  _)/ _  )/ ___)
+| |   | |_| | |   | |_( (/ /| |
+|_|    \___/|_|    \___)____)_|
+
+the conduit between web apps and the nucypher network
```

Comment:
Not very catchy 😆

Reply:
✔️ - temporarily updated. Can be changed later on if needed.